src/index.js

```diff
@@ -1 +1,20 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import displayScores from './modules/displayScores.js';
+import { postScore } from './modules/fetchData.js';
+
+const refresBtn = document.querySelector('.refresh-btn');
+refresBtn.addEventListener('click', () => {
+  displayScores();
+});
+
+const submitBtn = document.querySelector('.form');
+submitBtn.addEventListener('submit', async (e) => {
+  e.preventDefault();
+
+  await postScore();
+  document.getElementById('user').value = '';
+  document.getElementById('score').value = '';
+  displayScores();
```

Comment:
- [OPTIONAL] You have done a great job in implementing the changes requested by the previous reviewer, :clap: , However, as part of the required which says: `Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON)` ,  The scores should display when you click on the `REFRESHED` button but in your case, its automatically update when you click the `SUBMIT` button. Please resolve this to improve UX experience. :slightly_smiling_face: 

- Possible solution would be to remove the `displayScores();` method in line 17 and update your `refreshBtn` event listener like the example below: :arrow_down: 

`refresBtn.addEventListener('click', async (e) => {`
 ` e.preventDefault();`
 ` displayScores();`
`});`